.github/workflows/main.yml

```diff
@@ -78,8 +78,9 @@ jobs:
               PRIMARY: "substratevm"
           - env:
               JDK: "labsjdk-ce-11"
-              GATE: "build,debuginfotest"
+              GATE: "build,debuginfotesti,hellomodule"
```

Comment:
You changed gate tag from `debuginfotest` to `debuginfotesti`. Since no `debuginfotesti` actually exists those debuginfo tests are not run anymore. I suggest adding a new gate instead on piggybacking on `GATE: "build,debuginfotest"` (also because we are modifying the environment with `USE_NATIVE_IMAGE_JAVA_PLATFORM_MODULE_SYSTEM`. Other gate tags might not like that.

Reply:
Corrected, it must have been a typo. I also agree a separate gate is better - I added it as well.